sample/src/main/java/com/leinardi/android/speeddial/sample/MainActivity.java

```diff
@@ -55,16 +56,21 @@ private void initSpeedDial(boolean addActionItems) {
                     .ic_link_white_24dp)
                     .create());
 
-            Drawable drawable = AppCompatResources.getDrawable(MainActivity.this, R.drawable.ic_custom_color);
-            mSpeedDialView.addActionItem(new SpeedDialActionItem.Builder(R.id.fab_custom_color, drawable)
-                    .setFabBackgroundColor(ResourcesCompat.getColor(getResources(), R.color.material_white_1000,
-                            getTheme()))
-                    .setFabImageTintColor(ResourcesCompat.getColor(getResources(), R.color.inbox_primary, getTheme()))
-                    .setLabel(R.string.label_custom_color)
-                    .setLabelColor(Color.WHITE)
-                    .setLabelBackgroundColor(ResourcesCompat.getColor(getResources(), R.color.inbox_primary,
-                            getTheme()))
-                    .create());
+                Drawable drawable = AppCompatResources.getDrawable(MainActivity.this, R.drawable.ic_custom_color);
+                FabWithLabelView fabWithLabelView = mSpeedDialView.addActionItem(new SpeedDialActionItem.Builder(R.id
+                        .fab_custom_color, drawable)
+                        .setFabImageTintColor(ResourcesCompat.getColor(getResources(), R.color.inbox_primary, getTheme()))
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
**Severity**: ERROR
**File**: /home/travis/build/leinardi/FloatingActionButtonSpeedDial/sample/src/main/java/com/leinardi/android/speeddial/sample/MainActivity.java L62

Line is longer than 120 characters (found 122).


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-810533692>*